tests/32-action_options.sh

```diff
@@ -54,7 +54,7 @@ stest 0 mkdir -p "$TESTDIR"
 stest 0 touch "$TESTDIR/testfile"
 stest 0 singularity exec --home "$TESTDIR" "$CONTAINER" test -f "$TESTDIR/testfile"
 stest 0 singularity exec --home "$TESTDIR:/home" "$CONTAINER" test -f "/home/testfile"
-if [ -n "${SINGULARITY_OVERLAY_FS:-}" ]; then
+if [  "$SINGULARITY_OVERLAY_FS" = "1" ]; then
```

Comment:
Why this change?  Seems like the original line is more robust right?

Reply:
The original line doesn't work..

> Fix SINGULARITY_OVERLAY_FS check - this variable is always defined, 0 or 1 - so check for that, not -n

In `test.sh.in` that variable is always getting defined to 0 or 1... so the original test for nonzero length always has the same outcome whether or not overlay is available. Here's the test.sh.in bit:

```bash
SINGULARITY_OVERLAY_FS="0"
KVERS=`uname -r`
if test -f "/lib/modules/$KVERS/modules.dep"; then
    if grep -q 'overlay.ko' "/lib/modules/$KVERS/modules.dep"; then
        SINGULARITY_OVERLAY_FS="1"
    fi
fi
```

Reply:
Ah.  Good catch.